src/App.vue

```diff
@@ -1,32 +1,46 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+    <div id="app">
+        <div id="nav">
+            <router-link to="/">Home</router-link>
+            |
+            <router-link to="/about">About</router-link>
+        </div>
+        <el-row :gutter="24" justify="center" align="top">
+            <el-col :offset="5" :span="14" id="view">
+                <router-view/>
+            </el-col>
+        </el-row>
     </div>
-    <router-view/>
-  </div>
 </template>
 
 <style>
+html {
+    background-color: rgba(255, 211, 65, 0.4)
+}
+
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
+    font-family: Avenir, Helvetica, Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-align: center;
+    color: #2c3e50;
 }
 
 #nav {
-  padding: 30px;
+    padding: 30px;
 }
 
 #nav a {
-  font-weight: bold;
-  color: #2c3e50;
+    font-weight: bold;
+    color: #2c3e50;
 }
 
 #nav a.router-link-exact-active {
-  color: #42b983;
+    color: #42b983;
+}
+
+#view {
+    background-color: rgba(244,200,63,0.42);
+    border: 5px outset rgba(255, 211, 65, 0.75);
 }
```

Comment:
Move styles to an `assets` or `styles` directory, Create different style files for navbar, full site etc